### ISSUE TITLE
[SciPy 2018] Cannot find python modules when launching jupyter notebook

### ISSUE THREAD

**Issue by pushingrandomkeys** [ericmjl/Network-Analysis-Made-Simple#58]
I've set up the conda environment with the packages listed on the github page, but when I go to launch the Jupyter notebook, I get:

```
(Sci2018PyNetworkAnalysis) eric@mercury ~/SciPy 2018 Materials/Network-Analysis-Made-Simple/Network-Analysis-Made-Simple $ jupyter notebook
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fad11a0b740 (most recent call first):
Aborted
```

**Comment by ericmjl** [comment]
Thanks for the cross-post here, @pushingrandomkeys. This will provide a public channel for other participants to learn about this issue.

I'd suggest using the [Binder option](https://mybinder.org/v2/gh/ericmjl/Network-Analysis-Made-Simple/master); this should simplify setup for you. 

Also happy to do office hours evening of Sunday at the conference centre hotel to help debug.

**Comment by pushingrandomkeys** [comment]
Thanks @ericmjl 
For reference, OS is Ubuntu 14.04

**Comment by pushingrandomkeys** [comment]
Did some more playing and figured this out. I was using the "Manual Build" instructions (https://github.com/ericmjl/Network-Analysis-Made-Simple#manual-build) with conda, and jupyter isn't listed as one of the packages to install, so I didn't install it.

Since the error wasn't anything along the lines of "command not found", I figured somehow jupyter was installed - maybe outside of conda, or maybe brought in as a dependency of one of the packages I had installed. Apparently, I just had to install jupyter in the conda env. Still no idea why it did anything when I ran "jupyter notebook", rather than just say jupyter isn't installed and then quit.

Would suggest adding jupyter to the list of packages in the Manual Build section.
